pkg/systemd/overview-cards/healthCard.jsx

```diff
@@ -81,8 +77,10 @@ export class HealthCard extends React.Component {
                         <li>
                             <div role="group" className="system-information-updates">
                                 <div>
-                                    {this.state.systemInfoUpdatesIconVisible && <span id="system_information_updates_icon" className={this.state.systemInfoUpdatesIconClass} />}
-                                    <span id="system_information_updates_text" />
```

Comment:
check-packagekit uses these two removed IDs a lot, so they need to be adjusted to this. (Maybe a subsequent commit does that already, but the corresponding test updates should be in the same commit)

Reply:
Put back the IDs.